syntax = "proto3";

package response_models;

import "player-game-session-stats-response-model.proto";

message GameSessionStatsResponseModel {
	int32 first_team_kills = 1;
	int32 second_team_kills = 2;
	repeated response_models.PlayerGameSessionStatsResponseModel players_stats = 3;
}